torch/_tensor_docs.py

```diff
@@ -3619,11 +3684,33 @@ def callable(a, b) -> number
 See :func:`torch.topk`
 """)
 
+add_docstr_all('to_dense',
+               r"""
+to_dense() -> Tensor
+
+Returns a dense copy of the :attr:`self` tensor. Here, a dense tensor
```

Comment:
If we're going with "strided" let's be consistent and say "Creates a strided copy of self."

Reply:
fixed.